README/ReleaseNotes/v632/index.md

```diff
@@ -48,7 +49,18 @@ The following people have contributed to this new version:
 
 
 ## TTree Libraries
+### Add files from subdirectories with `TChain::Add` wildcarding
```

Comment:
Did we add wildcarding or globbing?

Reply:
Hi Philippe! You're right, globbing is the more accurate term. Changed it :+1: